auth-web/src/services/keycloak.services.ts

```diff
@@ -0,0 +1,58 @@
+import Keycloak from 'keycloak-js'
+import configHelper from '../util/config-helper'
+import { UserInfo } from '../models/userInfo'
+
+const kc: Keycloak.KeycloakInstance = null
+
+export default {
+
+  init (idpHint : string) {
+    this.cleanupSession()
+    let token = configHelper.getFromSession('KEYCLOAK_TOKEN')
+    this.kc = Keycloak(`/${process.env.VUE_APP_PATH}/config/keycloak.json`)
+    let kcLogin = this.kc.login
+    this.kc.login = (options) => {
+      options.idpHint = idpHint
+      kcLogin(options)
+    }
+
+    return this.kc.init({ token: token, onLoad: 'login-required' })
+  },
+
+  initSessionStorage () {
+    configHelper.addToSession('KEYCLOAK_TOKEN', this.kc.token)
```

Comment:
if possible use a string constant or something for KEYCLOAK_TOKEN , so that if it changes , we dont have to change it everywhere

Reply:
Done. A new constants file and an enum for Session Storage is created